pkg/debuggers/client.go

```diff
@@ -32,10 +33,24 @@ func RunSquashClient(debugger func(string) Debugger, conttopid platforms.Contain
 		return err
 	}
 
-	// TODO(mitchdraft) - the debug handler will need to be spawned/removed when a new debugging session is opened/closed
-	// The debug initialization request will include the watchNamespace. For now, use a known placeholder
-	watchNamespace := options.SquashClientNamespace
-	return NewDebugHandler(ctx, watchNamespace, daClient, debugger, conttopid).handleAttachments()
+	var kubeResClient *kubernetes.Clientset
+	if inClusterMode {
```

Comment:
why do you care if we are in cluster or not?

Reply:
I'll clean this up with #68 - I think go-utils/kubeutils takes this concern away